### ISSUE TITLE
Discretization or not in the evaluation ?

### ISSUE THREAD

**Issue by Sunarker** [ericjang/gumbel-softmax#5]
Hi, Eric, I extent this code for evaluation. But I found the results in this paper can not be achieved if I discretized the output in the evaluation. Instead, if I keep the training and test consistent, the results are similar to those in the paper.
